clients/command-pubsub/src/test/java/org/eclipse/hono/client/command/pubsub/PubSubBasedCommandTest.java

```diff
@@ -165,7 +218,7 @@ public void testFromMessageFailsForMessageWithoutSubject() {
     }
 
     private PubsubMessage getPubSubMessage(final String tenantId, final String deviceId, final String subject,
-            final String correlationId, final String responseRequired, final String via) {
+            final String correlationId, final String responseRequired, final String ackRequired, final String via) {
```

Comment:
instead of having to add a `null` parameter to the existing invocations, can we simply overload the method?